docs/conf.py

```diff
@@ -27,9 +27,9 @@
 author = "Qiskit Development Team"
 
 # The short X.Y version
-version = "0.44"
+version = "0.25"
 # The full version, including alpha/beta/rc tags
-release = "0.44.1"
+release = "0.25.2"
```

Comment:
This should be 0.44 and 0.44.2, because it's the version of the `qiskit` documentation.

Reply:
oh! that makes sense. Fixed in 7d736dab340236f08c917a72613cbabd9236ab3f